tests/metrics/network/network-metrics-memory-1g.sh.in

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures Proportional Set Size memory while running an
+#  inter (docker<->docker) 1Gbps network bandwidth using nuttcp
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Currently default nuttcp has a bug
+# see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=745051
+# Image name
+image=gabyct/nuttcp
+# This is required in order to reduce standard deviation
+total_time=6
+# This time (seconds) is required when
+# server and client are more stable, we need to
+# have server and client running for sometime and we
+# need to avoid to measure at the beginning of the running
+middle_time=3
+
+# Rate limit (speed at which transmitter send data, megabytes)
+rate_limit=10000
```

Comment:
Can you explain how this 10,000 translates to the '1Gbps' - I'm not sure I can figure it out?